github-webhook.js

```diff
@@ -227,10 +231,18 @@ function handleRules (logStream, rules, event) {
       }
     })
 
+    cp.stdout.pipe(past)
+    prefixStream(cp.stderr, '! ').pipe(past)
+
     if (logStream) {
-      prefixStream(cp.stdout, 'stdout: ').pipe(logStream, { end: false })
-      prefixStream(cp.stderr, 'stderr: ').pipe(logStream, { end: false })
+      past.pipe(logStream, {end: false})
     }
+    if (rule.report) getStream(past).then(function(str) {
```

Comment:
needs a `{` after the `if`

also, this should be exclusive to the `logStream` `if` above, we can't have the stream being piped into two places unless we introduce another utility that will allow it to be double-piped. It's either one or the other or something funky in between.

I also would rather not introduce `get-stream`, it's not a package I'm aware of and its usage here doesn't make me entirely comfortable -- there's no promises in this library yet so I'd like to avoid that for now, and there's no `catch` on this either. I'd rather use `bl`, which I'm obviously more comfortable with and it'll do the job just fine with a callback, like `past.pipe(bl((data, err) => { err or data.toString() ... }`

Reply:
Switched to bl (i like bl, i don't know why i used get-stream).
However i don't understand the comment piping in two places: i thought that was okay.